driver/gl/gl.go

```diff
@@ -24,9 +24,12 @@ import (
 )
 
 var textures = make(map[fyne.CanvasObject]uint32)
+var texturesMutex sync.RWMutex
 var refreshQueue = make(chan fyne.CanvasObject, 1024)
 
 func getTexture(object fyne.CanvasObject, creator func(canvasObject fyne.CanvasObject) uint32) uint32 {
+	texturesMutex.Lock()
```

Comment:
I'm pretty sure that the texture cache is only ever used from the main thread - did this change fix another bug?